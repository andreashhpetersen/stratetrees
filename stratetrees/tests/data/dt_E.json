{
    "variables": ["x","y"],
    "actions": ["A","B","C"],
    "root": {
        "var": "x",
        "bound": 2,
        "low": {
            "var": "y",
            "bound": 4,
            "low": {
                "var": "y",
                "bound": 2,
                "low": {
                    "action": 1
                },
                "high": {
                    "action": 0
                }
            },
            "high": {
                "action": 1
            }
        },
        "high": {
            "var": "y",
            "bound": 3,
            "low": {
                "var": "y",
                "bound": 1,
                "low": {
                    "action": 2
                },
                "high": {
                    "var": "x",
                    "bound": 5,
                    "low": {
                        "var": "x",
                        "bound": 3,
                        "low": {
                            "action": 0
                        },
                        "high": {
                            "action": 0
                        }
                    },
                    "high": {
                        "var": "x",
                        "bound": 7,
                        "low": {
                            "action": 0
                        },
                        "high": {
                            "action": 1
                        }
                    }
                }
            },
            "high": {
                "var": "y",
                "bound": 5,
                "low": {
                    "var": "x",
                    "bound": 6,
                    "low": {
                        "var": "x",
                        "bound": 4,
                        "low": {
                            "action": 0
                        },
                        "high": {
                            "action": 0
                        }
                    },
                    "high": {
                        "action": 0
                    }
                },
                "high": {
                    "action": 2
                }
            }
        }
    }
}
