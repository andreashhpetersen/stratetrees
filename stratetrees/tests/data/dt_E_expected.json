{
    "commment": "this is not an optimal partitioning, but it is a correct one (it is what comes when numpy is seeded with 42)",
    "variables": ["x","y"],
    "actions": ["A","B","C"],
    "root": {
        "var": "x",
        "bound": 2,
        "low": {
            "var": "y",
            "bound": 4,
            "low": {
                "var": "y",
                "bound": 2,
                "low": {
                    "action": 1
                },
                "high": {
                    "action": 0
                }
            },
            "high": {
                "action": 1
            }
        },
        "high": {
            "var": "y",
            "bound": 1,
            "low": {
                "action": 2
            },
            "high": {
                "var": "y",
                "bound": 5,
                "low": {
                    "var": "x",
                    "bound": 6,
                    "low": {
                        "action": 0
                    },
                    "high": {
                        "var": "x",
                        "bound": 7,
                        "low": {
                            "action": 0
                        },
                        "high": {
                            "var": "y",
                            "bound": 3,
                            "low": {
                                "action": 1
                            },
                            "high": {
                                "action": 0
                            }
                        }
                    }
                },
                "high": {
                    "action": 2
                }
            }
        }
    }
}
